### DISCUSSION TITLE
Missing some icons

### DISCUSSION THREAD

**Discussion by nikobonomi** [nvim-tree/nvim-tree.lua#2594] [Q&A]
### Description

I'm having issues with some icons, namely some of git.  

I'm on the last version of nvim-tree (installed yesterday) as well as nvim-web-devicons.

As font I'm using FiraCode 3.1.1 from nerd fonts. (tried also with the Hack font)
(also tried v3.0.0 and pre 3.0)

As you can see from the help (: h NvimTree), I'm missing some some git icons, the symlink arrow and the bookmark.

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/9111900/5578d4e7-71c2-421d-b9eb-a630456cc722)

Not really sure how to proceed (I'm new to nvim)

As for now from the conf I have overwritten some glyphs, but I would like to have it working with the standard ones.

### Neovim version

```text
NVIM v0.10.0-dev-1836+g36552adb3
Build type: RelWithDebInfo
LuaJIT 2.1.1702233742
```

### Operating system and version

Ubuntu

### Windows variant

WSL

### nvim-tree version

50f30bc

### Clean room replication

```lua
Nothing special to do, just a simple config

-- disable netrw at the very start of your init.lua
vim.g.loaded_netrw = 1
vim.g.loaded_netrwPlugin = 1
--
-- -- set termguicolors to enable highlight groups
vim.opt.termguicolors = true
--
-- -- empty setup using defaults
require("nvim-tree").setup {
    renderer = {
        icons = {
            glyphs = {
                git = {
                    untracked = "",
                    unstaged = "U",
                    renamed = "R",
                    deleted = "D",
                    ignored = "I"
                }
            }
        }
    }
}
```

### Steps to reproduce

:h NvimTree

Scroll to the demo config

### Expected behavior

To have the correct icons showing

**Comment by geril07** [comment]
I've installed `FiraCode Nerd Font` from https://www.nerdfonts.com/font-downloads (I guess it 3.1.1), for me using kitty terminal, seems to work fine. Can you make sure you are running latest nerd font?

**Comment by nikobonomi** [comment]
I'm sure since I've downloaded the font again this morning from that exact page :/

Could it be somethings with the WSL console on windows?

**Comment by geril07** [comment]
Which terminal do you use? I think output should be the same inside wsl and outside. Can you try outside of wsl?

**Comment by nikobonomi** [comment]
Just plain WSL, means I open ubuntu from the start menu.
Then using plain bash I open nvim.

I will spin up an ubuntu vbox and see there

**Comment by gegoune** [comment]
This is not an issue of nvim-tree so converting to discussion, but it will most likely be due to missing glyphs or misconfigured terminal emulator.

**Comment by nikobonomi** [comment]
Just tested on Ubuntu inside a vBox and it works :)

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/9111900/b38dfb11-c329-42a1-871f-f23f89424a75)

So maybe windows has something to do whit this.
